package.json

```diff
@@ -45,6 +45,7 @@
     ]
   },
   "dependencies": {
+    "acorn": "^7.3.1",
```

Comment:
Why this commit adds `acorn`?

Reply:
it shouldn't be there.  I'll commit an updated version soon.